xclim/indices/_agro.py

```diff
@@ -141,13 +141,13 @@ def huglin_index(
 
     Parameters
     ----------
-    tasmin: xarray.DataArray
-      Minimum daily temperature.
+    tas: xarray.DataArray
+      Mean daily temperature.
     tasmax: xarray.DataArray
       Maximum daily temperature.
     lat: xarray.DataArray
       Latitude coordinate.
-    thresh_tasmin: str
+    thresh: str
       The minimum temperature threshold.
```

Comment:
```suggestion
      The temperature threshold.
```
I guess this has to change to? I have a hard time thinking of a better description... What is (Tmean + Tmax)/2?? Like the approximate day-time mean temperature? Maybe just removing "minimum" is ok.